run.nextflow.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+import os
+import subprocess
+import sys
+import time
+
+# DIR where the current script resides
+DIR = os.path.dirname(os.path.abspath(__file__))
+
+# set default values for arguments if not provided

```

Comment:
Please consider argparse.ArgumentParser. It has more sophesticated input checking. I used it before and liked it very much.